kafka-ui-react-app/src/components/ConsumerGroups/__test__/ConsumerGroups.spec.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { clusterConsumerGroupsPath } from 'lib/paths';
+import {
+  screen,
+  waitFor,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import ConsumerGroups from 'components/ConsumerGroups/ConsumerGroups';
+import { consumerGroups } from 'redux/reducers/consumerGroups/__test__/fixtures';
+import { render } from 'lib/testHelpers';
+import fetchMock from 'fetch-mock';
+import { Route } from 'react-router';
+
+const clusterName = 'cluster1';
+
+const renderComponent = () =>
+  render(
+    <Route path={clusterConsumerGroupsPath(':clusterName')}>
+      <ConsumerGroups />
+    </Route>,
+    {
+      pathname: clusterConsumerGroupsPath(clusterName),
+    }
+  );
+
+describe('ConsumerGroup', () => {
+  afterEach(() => {
+    fetchMock.reset();
+  });
+
+  it('renders with initial state', async () => {
+    renderComponent();
+
+    expect(screen.queryByRole('progressbar')).toBeInTheDocument();
```

Comment:
Replacing queryBy with getBy since it exist in Dom to pin point the point of failure.